pkg/util/trivy/containerd.go

```diff
@@ -0,0 +1,172 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package trivy
```

Comment:
I think it'd be good to add a small comment clarifying what has been copied from the trivy library and what has been adapted. It might be useful later if we need to update the trivy version.